zeroeq/http/server.cpp

```diff
@@ -26,6 +32,30 @@ std::string _toLower( std::string value )
     return value;
 }
 
+// http://stackoverflow.com/questions/5343190/how-do-i-replace-all-instances-of-a-string-with-another-string
+std::string _replaceAll( std::string subject, const std::string& search,
+                         const std::string& replace )
+{
+    size_t pos = 0;
+    while( (pos = subject.find( search, pos )) != std::string::npos )
+    {
+         subject.replace( pos, search.length(), replace );
+         pos += replace.length();
+    }
```

Comment:
Because it is one. Precondition is before while, loop statement is at the end of the while scope. Not important, though.